### PR TITLE
Add eZ Platform 2.5.5

### PR DESCRIPTION
**Pull request by SylvainGuittard**
Template for eZ Platform Open Source Edition (v2.5.5)

### CODE REVIEW

File: templates/ezplatform.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    # Unique machine name, prefaced by a vendor or organization identifier
+    id: ezsystems/ezplatform
+    # The human-readable name of the template.
+    name: eZ Platform 2.5.5
+    # Human-readable descriptive text for the template. Supports limited HTML.
+    description: The latest version of <b>eZ Platform</b> without customization. Perfect to start a new project.
+    # A list of tags associated with the template.
+    tags:
+        - PHP
+        - eZ
```
Comment (Crell) [inline templates/ezplatform.yaml]:
```suggestion
        - eZ Platform
```

File: templates/ezplatform.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        - CMS
+        - Symfony
```
Comment (Crell) [inline templates/ezplatform.yaml]:
```suggestion
```

File: templates/ezplatform.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+version: 1
+
+# Templates are a small amount of information supporting a template URL. Each template is selectable at the project-creation step.
+info:
+    # Unique machine name, prefaced by a vendor or organization identifier
+    id: ezsystems/ezplatform
+    # The human-readable name of the template.
+    name: eZ Platform 2.5.5
+    # Human-readable descriptive text for the template. Supports limited HTML.
+    description: The latest version of <b>eZ Platform</b> without customization. Perfect to start a new project.
```
Comment (Crell) [inline templates/ezplatform.yaml]:
```suggestion
    description: The latest version of <b>eZ Platform</b> without customization. Good for starting a new project.
```

File: templates/ezplatform.yaml (hunk 2 above)
Comment (SylvainGuittard) [reply templates/ezplatform.yaml]: Thanks @Crell 
I do have a question about this suggestion. I can see that `Akeneo` and `Pimcore` listed with Symfony. `eZ Platform` is also using Symfony fullstack. Is there a rule to be able to have `Symfony`?

Comment (Crell) [reply templates/ezplatform.yaml]: Hm, you're right, we are tagging other Symfony-based apps.  I thought we weren't.  Ignore this one.

**Comment by SylvainGuittard** [comment]
Thanks again @Crell 🎉

**Pull request merged**